### PR TITLE
Document 'feedback' parameters

### LINKED ISSUE #1731: [2pt] Configuration update, allow to disable sending crash information
**Product:** Tarantool
**Since:** 2.7.1
**Audience/target:** 
**Root document:** https://www.tarantool.io/en/doc/latest/reference/configuration/#feedback
**SME:** @cyrillos 
**Peer reviewer:** @

For a better analysis of program crashes the information associated with
the crash such as

 - utsname (similar to `uname -a` output except the network name)
 - build information
 - reason for a crash
 - call backtrace

is sent to the feedback server. To disable it set `feedback_crashinfo`
to `false`.
Requested by @cyrillos in https://github.com/tarantool/tarantool/commit/f132aa9b95b86e46f0ae74a9f8c115957b040f55.

# Definition of done

- [ ] new `feedback_crashinfo` config
- [ ] crash info data format

Related to #1749

### LINKED ISSUE #4019: [Config] Document 'feedback' configuration settings
**Related dev issues:** https://github.com/tarantool/tarantool/issues/8192, https://github.com/tarantool/tarantool/issues/5261, https://github.com/tarantool/tarantool/issues/5589
**Related doc issues:** https://github.com/tarantool/doc/issues/1731, https://github.com/tarantool/doc/issues/1749
**PRs:** https://github.com/tarantool/tarantool/pull/8231/commits/c4c0b5d068d80c8edf3a6cdbcaf2ba63f028651f

**Product:** Tarantool
**Since:** 3.0
**Root document:** 
- A new `feedback` section in the [Configuration reference](https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/).

**SME:** @ Totktonada

# Details

Options:
- https://github.com/tarantool/tarantool/blob/f58bfc97c627f1838696697339fb3e443c05767a/src/box/lua/config/instance_config.lua#L1652

Related topics:
- https://www.tarantool.io/en/doc/latest/reference/configuration/#feedback

### PR DESCRIPTION
**Pull request by andreyaksenov**
Fixes https://github.com/tarantool/doc/issues/4019.

Staging: https://docs.d.tarantool.io/en/doc/config-feedback/reference/configuration/configuration_reference/#feedback

### CODE REVIEW

File: doc/reference/configuration/configuration_reference.rst [hunk 1]
```diff
@@ -1671,6 +1671,114 @@ See also: :ref:`supervised_failover_overview_fault_tolerance`
...
+    |
+    | Type: string
+    | Default: https://feedback.tarantool.io
+    | Environment variable: TT_FEEDBACK_HOST
+
+..  _configuration_reference_feedback_interval:
+
+..  confval:: feedback.interval
+
+    The interval (in seconds) of sending information.
```
Comment (Totktonada) [inline doc/reference/configuration/configuration_reference.rst:1739]: Does the 'X is the interval of sending in seconds' always means 'X means sending data every X seconds'? Can it be interpreted like 'how many seconds may be consumed to transmit the data over the network'?

The same question about the metrics collect interval.

**Review by Totktonada — APPROVED** [review]
Thanks! Looks good for me.

File: doc/reference/configuration/configuration_reference.rst (hunk 1 above)
Comment (andreyaksenov) [reply doc/reference/configuration/configuration_reference.rst:1739]: > Can it be interpreted like 'how many seconds may be consumed to transmit the data over the network'?

I believe it cannot be interpreted like this. @lenkis what do you think?

Comment (lenkis) [reply doc/reference/configuration/configuration_reference.rst:1739]: I'd never think about this kind of interpretation, really. I'd leave the phrase as is.

**Review by lenkis — APPROVED** [review]

File: doc/reference/configuration/configuration_reference.rst (hunk 1 above)
Comment (Totktonada) [reply doc/reference/configuration/configuration_reference.rst:1739]: No problems. Thanks for taking care :)

**Pull request merged**